pr-data.csv

```diff
@@ -3133,7 +3133,7 @@ https://github.com/kevinsawicki/http-request,2d62a3e9da726942a93cf16b6e91c0187e6
 https://github.com/kevinsawicki/http-request,2d62a3e9da726942a93cf16b6e91c0187e6c0136,lib,com.github.kevinsawicki.http.HttpRequestTest.singleSslSocketFactory,OD-Vic,Opened,https://github.com/kevinsawicki/http-request/pull/151,
 https://github.com/kevinsawicki/http-request,2d62a3e9da726942a93cf16b6e91c0187e6c0136,lib,com.github.kevinsawicki.http.HttpRequestTest.singleVerifier,OD-Vic,Opened,https://github.com/kevinsawicki/http-request/pull/151,
 https://github.com/kevinsawicki/http-request,2d62a3e9da726942a93cf16b6e91c0187e6c0136,lib,com.github.kevinsawicki.http.HttpRequestTest.verifierAccepts,OD-Vic,Opened,https://github.com/kevinsawicki/http-request/pull/151,
-https://github.com/kiegroup/optaplanner,dff7457cb5ba1557727deddef5b675dd00fbf8a8,optaplanner-core,org.optaplanner.core.config.solver.SolverConfigTest.xmlConfigFileRemainsSameAfterReadWrite,ID,,,
+https://github.com/kiegroup/optaplanner,dff7457cb5ba1557727deddef5b675dd00fbf8a8,optaplanner-core,org.optaplanner.core.config.solver.SolverConfigTest.xmlConfigFileRemainsSameAfterReadWrite,ID,MovedOrRenamed,,https://github.com/kiegroup/optaplanner/commit/4dd2dc8700a1972ba3148e2f3fabe5568f0b0c69
```

Comment:
What script did you use to find that the test was MovedOrRenamed? Where is the new location? README says to add a new location for each such test.

Reply:
It's the module that got renamed not the test though, if you think this change should not be considered as MovedOrRenamed I can remove it.

Reply:
Updated, also the test `com.spotify.dbeam.args.JdbcExportArgsTest.should parse correctly for mysql connection` is deleted. Neither test file or test suite exists now.